README.md

```diff
@@ -417,7 +417,7 @@ For many examples on how to use WDL see [the WDL site](https://github.com/broadi
 
 # Configuring Cromwell
 
-Cromwell's default configuration file is located at `src/main/resources/application.conf`.
+Cromwell's default configuration file is located at `core/src/main/resources/reference.conf`.
```

Comment:
If I read this as-is, I'd assume that anything in `reference.conf` was going to be used by Cromwell.
I'll leave it up to you (or our recently-returned travel-writer @katevoss) whether to change it